components/inputs/test/input-radio.visual-diff.js

```diff
@@ -0,0 +1,48 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-input-radio', () => {
+
+	const visualDiff = new VisualDiff('input-radio', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		await page.setViewport({width: 800, height: 800, deviceScaleFactor: 2});
+		await page.goto(`${visualDiff.getBaseUrl()}/components/inputs/test/input-radio.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	after(() => browser.close());
+
+	beforeEach(async() => {
+		await visualDiff.resetFocus(page);
+	});
+
+	async function getRadio(id, val) {
```

Comment:
I think the `async` and the `await` are not needed here.  The `getRadio` is returning a promise provided by `evaluateHandle`, and the consumer of `getRadio` is in control of whether to wait or not (which it is).